dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAPUnitInfo.java

```diff
@@ -58,6 +58,8 @@
 
     private static final TraceComponent tc = Tr.register(JPAPUnitInfo.class, JPA_TRACE_GROUP, JPA_RESOURCE_BUNDLE_NAME);
     private static final TraceComponent tcTransformer = Tr.register(CLASS_NAME + "_Transformer", JPAPUnitInfo.class, JPA_TRACE_GROUP + ".Transformer", JPA_RESOURCE_BUNDLE_NAME);
+    private static final TraceComponent tcBytecodeCapture = Tr.register(CLASS_NAME + "_BytecodeCapture", JPAPUnitInfo.class, JPA_TRACE_GROUP + ".BytecodeCapture",
+                                                                        JPA_RESOURCE_BUNDLE_NAME);
```

Comment:
Trace group is JPA.BytecodeCapture, and not JPA_BytecodeCapture as described in PR description?

Reply:
You're right, I'll update the PR description, though `JPA_BytecodeCapture` also works.

Reply:
So I've changed the PR description back to `JPA_BytecodeCapture`.  Maybe there's some RAS translating going on somewhere.